### PR TITLE
KEYCLOAK-16679 Add algorithm settings for client assertion signature in OIDC identity broker

### PR DESCRIPTION
**Pull request by i7a7467**
https://issues.redhat.com/browse/KEYCLOAK-16679

### CODE REVIEW

**Comment by hmlnarik** [comment]
Code PR: keycloak/keycloak#7691

**Comment by mposolda** [comment]
This PR looks good to me and from my point of view, it can be merged once the codebase PR is merged.

@andymunro Could you please also review this PR?

File: server_admin/topics/identity-broker/oidc.adoc
Comment (andymunro) [inline server_admin/topics/identity-broker/oidc.adoc]: remove "the" before both occurrences of "in"

**Review by andymunro — CHANGES_REQUESTED** [review]
Just one line to change. Thanks.

File: server_admin/topics/identity-broker/oidc.adoc
Comment (i7a7467) [reply server_admin/topics/identity-broker/oidc.adoc]: @mposolda @andymunro 
Thanks. I fixed your review point.

**Review by mposolda — APPROVED** [review]
@ryandawsonuk @andymunro Thanks for the PR and for the review!

**Pull request merged**
